app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -441,4 +441,13 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- OptionAudioLanguage -->
+  <dimen name="option_audio_language_padding_start">16dp</dimen>
+  <dimen name="option_audio_language_padding_end">16dp</dimen>
+
+  <!--TopicLessonsSummary-->
```

Comment:
@shankarpriyank write this as ` <!-- TopicLessonsSummary -->` do this for all dimen files

Reply:
Done